webportal/src/app/utils/pai_api.js

```diff
@@ -3,6 +3,27 @@
 import querystring from 'query-string';
 import yaml from 'js-yaml';
 
+const REST_SERVER_URI = process.env.REST_SERVER_URI;
```

Comment:
rest_server_url could be get from context, and no need to set an extra env variable, see https://github.com/microsoft/openpaimarketplace/blob/3718aeb25ee4c5ddbdb52570eb4654de31bd6ad5/webportal/src/app/app.jsx#L47